internal/gps/vcs_source.go

```diff
@@ -47,6 +47,10 @@ func (bs *baseVCSSource) getManifestAndLock(ctx context.Context, pr ProjectRoot,
 	}
 
 	m, l, err := an.DeriveManifestAndLock(bs.repo.LocalPath(), pr)
+	if os.IsNotExist(err) {
+		return prepManifest(nil), nil, nil
```

Comment:
@sdboyer if I remove the special case for `os.IsNotExist` then a slew of tests fail in cmd/dep.

http://paste.ubuntu.com/24590545/

I wonder if something is messed up with the logic that lets this path get through to this point. Or maybe the test fixtures are borken?